noise/README.md

```diff
@@ -291,6 +293,24 @@ internal libp2p data.
 
 The XX handshake MUST be supported by noise-libp2p implementations.
 
+### Noise Extensions
+
+Since the Noise handshake pattern itself doesn't define any extensibility
+mechanism, this specification defines an extension registry, modeled after
+[RFC 6066](https://www.rfc-editor.org/rfc/rfc6066) (for TLS) and
+[RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000#section-19.21)
+(for QUIC).
+
+This spec currently defines 3 extension code points for the `NoiseExtensions`
```

Comment:
I only see one code in this spec (`repeated bytes webtransport_certhashes = 1;`), right? Or am I missing something?

Reply:
I removed the other ones and forgot to update this sentence.

Reply:
We probably shouldn't count the extensions here anyway. This is just guaranteed to get out of sync in the future. I removed it in 8e4b017c6a8f4251a45acf604f0a36e6bc81df23.